app/coins/btc.js

```diff
@@ -226,25 +226,25 @@ module.exports = {
 			alertBodyHtml: "This is one of 2 'duplicate coinbase' transactions. An early bitcoin bug (fixed by <a href='https://github.com/bitcoin/bips/blob/master/bip-0030.mediawiki'>BIP30</a>) allowed identical coinbase transactions - a newer duplicate would overwrite older copies. This transaction was the coinbase transaction for <a href='/block-height/91812'>Block #91,812</a> and, ~3 hours later, <a href='/block-height/91842'>Block #91,842</a>. The 50 BTC claimed as the coinbase for block 91,812 were also overwritten and lost."
 		}
 	],
-	exchangeRateData:{
-		jsonUrl:"https://api.coindesk.com/v1/bpi/currentprice.json",
+	exchangeRateDataUSD:{
+		jsonUrl:"https://api.liquid.com/products/618",
 		responseBodySelectorFunction:function(responseBody) {
 			//console.log("Exchange Rate Response: " + JSON.stringify(responseBody));
-
-			var exchangedCurrencies = ["USD", "GBP", "EUR"];
-
-			if (responseBody.bpi) {
-				var exchangeRates = {};
-
-				for (var i = 0; i < exchangedCurrencies.length; i++) {
-					if (responseBody.bpi[exchangedCurrencies[i]]) {
-						exchangeRates[exchangedCurrencies[i].toLowerCase()] = responseBody.bpi[exchangedCurrencies[i]].rate_float;
-					}
-				}
-
-				return exchangeRates;
+			if (responseBody.last_traded_price) {
+				let exchangeRate = responseBody.last_traded_price;
+				return exchangeRate;
+			}
+			return null;
+		}
+	},
+	exchangeRateDataBTC:{
+		jsonUrl:"https://api.liquid.com/products/619",
+		responseBodySelectorFunction:function(responseBody) {
+			//console.log("Exchange Rate Response: " + JSON.stringify(responseBody));
+			if (responseBody.last_traded_price) {
+				let exchangeRate = responseBody.last_traded_price;
```

Comment:
What about just returning `return responseBody.last_traded_price;` without setting temporary variable?

Reply:
Will be done.